src/onTTFB.ts

```diff
@@ -67,21 +67,28 @@ export const onTTFB = (onReport: ReportCallback, opts?: ReportOpts) => {
     const navEntry = getNavigationEntry();
 
     if (navEntry) {
-      // The activationStart reference is used because TTFB should be
-      // relative to page activation rather than navigation start if the
-      // page was prerendered. But in cases where `activationStart` occurs
-      // after the first byte is received, this time should be clamped at 0.
-      metric.value = Math.max(navEntry.responseStart - getActivationStart(), 0);
+      const responseStart = navEntry.responseStart;
 
       // In some cases the value reported is negative or is larger
       // than the current page time. Ignore these cases:
       // https://github.com/GoogleChrome/web-vitals/issues/137
       // https://github.com/GoogleChrome/web-vitals/issues/162
-      if (metric.value < 0 || metric.value > performance.now()) return;
+      if (responseStart < 0 || responseStart > performance.now()) return;
 
-      metric.entries = [navEntry];
+      // If the navigation entry's `responseStart` value is 0, ignore it.
+      // This likely means the request included a cross-origin redirect, and
+      // the browser has removed timing info for privacy/security reasons.
+      // See: https://github.com/GoogleChrome/web-vitals/issues/275
+      if (responseStart > 0) {
```

Comment:
Why not change the test above to below to capture this case and avoid the need of this test?:

```js
if (responseStart <= 0 ||  || responseStart > performance.now()) return;
```

That wouldn't set up the onBFCacheRestore, but think that's OK since that is only called if a  `navigation` entry was reported for the initial load.

Reply:
Yeah, fair point. I did this to ensure the logic ran for bfcache restores, but I suppose it's an open question as to whether or not it should. Also, if it should then it arguably should for the above cases as well, so the early return isn't needed.

WDYT? Should we report zero values for bfcache restores in cases where the original TTFB value was never reported? Or should we just ignore those visits altogether?

I guess one argument for not reporting them is that bugs in Safari might make it look to the site owner that bfcache hit rates are much higher in Safari than in other browsers, when if fact that may not be the case.

Reply:
Hmmm… there’s argument for both sides.

I think I’m veering slightly on the don’t report if original nav didn’t have it.

Either way think it’s confusing at present with the comment below saying we don’t report in those cases if we do sometimes.

Reply:
Ok, I'll update to not report bfcache restores in these cases.